src/js/form-util.js

```diff
@@ -127,9 +128,21 @@ export function prepareInputs (formInputs, reasonInputs, reasonFieldset, reasonA
       return
     }
 
-    console.log(getProfile(formInputs), reasons)
+    const profile = getProfile(formInputs)
 
-    const pdfBlob = await generatePdf(getProfile(formInputs), reasons, pdfBase)
+    console.log(profile, reasons)
```

Comment:
Il faudrait peut être supprimer ça 😂

Reply:
C'était déjà là du coup dans le doute j'avais laissé. Mais j'ai retiré, on verra bien 🙂

Reply:
Je pense qu'il est risqué de stocker des données personnelles dans le localstorage.

- https://dev.to/rdegges/please-stop-using-local-storage-1i04


Peut-être utiliser les mécanismes déjà intégrés dans HTML5 et votre navigateur ?

- https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete